cmd/gardenlet/app/options.go

```diff
@@ -75,3 +79,41 @@ func (o *options) Validate() error {
 func (o *options) LogConfig() (string, string) {
 	return o.config.LogLevel, o.config.LogFormat
 }
+
+// syncBackupSecretRefAndCredentialsRef syncs the seed backup credentials when possible.
+// TODO(vpnachev): Remove this function after v1.121.0 has been released.
+func syncBackupSecretRefAndCredentialsRef(backup *gardencorev1beta1.SeedBackup) {
```

Comment:
Please move this function to https://github.com/gardener/gardener/blob/master/cmd/gardenlet/app/migration.go file.